README.md

```diff
@@ -28,10 +29,14 @@ permanent partition onto `/var`:
 ```
 mkdir /perm/container-storage
 mount --bind /perm/container-storage /var
+mkdir -p /var/tmp
 ```
 
 #### Run a container
 
+To have `podman` easily be able to find other binaries like `nsenter` and `iptables` simply add `/user` and `/usr/local/bin` to `PATH` before running `podman`.
```

Comment:
Are you using an older `breakglass`? It arranges for both of these to be in $PATH already

Reply:
Oh! Let me check

Reply:
Agreed, I checked that and it does take care of that.

I see after the latest breakglass /var/tmp is also not needed. I can close this pull request if that's ok.

Reply:
Is `/var/tmp` truly not needed or does it already exist now on your `/perm`? I’m not entirely clear on whether podman will create it or not, but `breakglass` doesn’t have anything to do with `/var/tmp` :)

Reply:
I think it was fixed because of TMPDIR variable which is introduced in latest versions of breakglass

https://github.com/gokrazy/breakglass/commit/564a0eceafb7532baf4d0cc51b75be95d47047d9

Reply:
Ah, yep 😅

We’ll need to add `TMPDIR` to gokrazy services itself, too — I was vaguely remembering having to deal with `/var/tmp`, too.

Yep, this PR can be closed then.